src/rest/RequestHandler.js

```diff
@@ -125,13 +125,7 @@ class RequestHandler {
         /**
          * Emitted when the client hits a rate limit while making a request
          * @event Client#rateLimit
-         * @param {Object} rateLimitInfo Object containing the rate limit info
-         * @param {number} rateLimitInfo.timeout Timeout in ms
-         * @param {number} rateLimitInfo.limit Number of requests that can be made to this endpoint
-         * @param {string} rateLimitInfo.method HTTP method used for request that triggered this event
-         * @param {string} rateLimitInfo.path Path used for request that triggered this event
-         * @param {string} rateLimitInfo.route Route used for request that triggered this event
-         * @param {boolean} rateLimitInfo.global Whether the rate limit that was reached was the global limit
+         * @param {RateLimitData} rateLimitData Object containing the rate limit info
```

Comment:
Make sure these have the same properties 🙏

Reply:
yup. [RateLimitData](https://github.com/discordjs/discord.js/blob/da6f21a3fca084f27d115754fed5fff8091cce77/src/util/Constants.js#L8) defined in Constants, has exactly the same props.